control-plane-operator/controllers/hostedcontrolplane/csi/kubevirt/files/controller.yaml

```diff
@@ -0,0 +1,108 @@
+kind: Deployment
```

Comment:
are all these yamls copied from somewhere else? if so can we reference the source so they can be synced and maintained?

Reply:
For the components running management side we set some labels to leverage affinity/antiaffiniy and also nodeSelector, see https://github.com/openshift/hypershift/blob/950ea390af0981824a2e953a537190666208ec09/support/config/deployment.go#L279-L292

Reply:
fixed, I apply the generic deployment config to the controller deployment now